### ISSUE TITLE
`deltamethod()` shortcuts should be `b1, b2` instead of `r1, r2`

### ISSUE THREAD

**Issue by vincentarelbundock** [vincentarelbundock/marginaleffects#380]

**Comment by grantmcdermott** [comment]
Yeah, the `b` suffix isn't perfect. But I think it has enough inertia that it brings your audience along without too much trouble.

I think option 1 (`b1`, `b2`) is perfectly fine, given if requires only a minor change.
